x-pack/plugins/apm/server/tutorial/envs/elastic_cloud.ts

```diff
@@ -22,13 +26,15 @@ import {
 } from '../instructions/apm_agent_instructions';
 import { CloudSetup } from '../../../../cloud/server';
 
-export function createElasticCloudInstructions(cloudSetup?: CloudSetup) {
+export function createElasticCloudInstructions(
+  cloudSetup?: CloudSetup
+): TutorialSchema['elasticCloud'] {
   const apmServerUrl = cloudSetup?.apm.url;
   const instructionSets = [];
 
-  if (!apmServerUrl) {
-    instructionSets.push(getApmServerInstructionSet(cloudSetup));
-  }
+  instructionSets.push(
+    getApmServerInstructionSet({ cloudSetup, hasApmServerUrl: !!apmServerUrl })
```

Comment:
Perhaps make the naming a little more explanatory. If there's no apm server url, that means apm server has not been enabled (yet).

```suggestion
    getApmServerInstructionSet({ cloudSetup, isApmServerEnabled: !!apmServerUrl })
```